README.md

```diff
@@ -249,3 +249,17 @@ The plugin has an API consisting of one required parameter and multiple optional
     -   woff - [ttf2woff](https://github.com/fontello/ttf2woff).
     -   eot - [ttf2eot](https://github.com/fontello/ttf2eot).
 -   See [webfonts-generator#formatoptions](https://github.com/vusion/webfonts-generator#formatoptions)
+
+## Public API
+
+The plugin exposes a public API that can be used inside another plugins using [Rollup inter-plugin communication mechanism](https://rollupjs.org/plugin-development/#inter-plugin-communication).
+
+Currently available methods:
+
+### getGeneratedWebfonts
+
+-   **returns**: `{ type: 'svg' | 'ttf' | 'woff2' | 'woff' | 'eot', href: string }`
+-   **description**
+    -   `type` - a font format generated by a plugin
+    -   `href` - a path to a generated font
+-   [This repo](https://github.com/stryaponoff/vite-plugin-preload-webfont) contains the usage example.
```

Comment:
I was trying to find a best way to give and example of how it works without adding a big code examples to the README.md and without duplicating your ./example dir so I just made another repo with plugin example.

Reply:
That works IMHO 🙂